contracts/seed/Seed.sol

```diff
@@ -207,14 +204,14 @@ contract Seed {
                   add(_fundingAmount) <= hardCap,
             "Seed: amount exceeds contract sale hardCap");
 
-        require( seedRemainder >= seedAmount && feeRemainder >= feeAmount,
+        require( seedRemainder >= seedAmount,
```

Comment:
Because, feeAmount and feeRemainder are both directly proportional to seedAmount and seedRemainder.